README.md

```diff
@@ -4,9 +4,9 @@ This is a living document that can serve as a resource for finding people includ
 
 This resource began in reponse to a discussion post started by [Christine Belzie](https://github.com/CBID2) titled [Where are all the women owned Open Source projects](https://dev.to/cbid2/where-are-all-the-women-owned-open-source-projects-4pd0). In looking for some resources to send Christine, I was unable to find anything thorough or currently maintained and decided to start this project.  It is our hope that this resource will get so large that we will have to find a better solution to hold all the amazing resources.
 
-Contributions to this resource are welcome and encouraged. 
+Contributions to this resource are welcome. ***NOTE**: To ensure that this repository stays true to its mission, we will prioritize open source projects and related content that is created by womxn,especially those from other marginalized communities.
```

Comment:
I don't think this is necessary, please remove and we can revisit the readme in the future if there seems to be confusion on the purpose of if.

Reply:
Sure

Reply:
It's already implied by the existence of the repo adn the rest of the description.  The intro is already quite verbose and there have been no indications there is confusion from submitters yet.  This issue can be revisited should confusion arise from contributors

Reply:
Ah okay.

Reply:
Yeah, totally understand your point but haven't seen confusion about the repo yet so I don't want to add bulk unless folks are coming to us with confusion. Appreciate the discussion always

Reply:
You're welcome! I just noticed that people tend to not mention womxn from other marginalized groups unless that disclaimer is there.

Reply:
it's in the first sentence of the readme so I think it's clear

Reply:
Oh no I was speaking for other women-centric initiatives and ranting a little lol. We're good 👍🏾